src/Hooks/CheckoutScriptHooks.php

```diff
@@ -82,7 +82,10 @@ private function getDeliveryOptionsPosition(): string
     {
         return Filter::apply(
             'deliveryOptionsPosition',
-            Settings::get(CheckoutSettings::DELIVERY_OPTIONS_POSITION, CheckoutSettings::ID)
+            Settings::get(
+                CheckoutSettings::DELIVERY_OPTIONS_POSITION,
+                CheckoutSettings::ID
+            ) ?? 'woocommerce_after_checkout_billing_form'
```

Comment:
deze string mag hier niet, en hier is al wat moois voor, zie `filterDefaults` in de bootstrapper